test/mail_ssl_manual.bats

```diff
@@ -22,20 +22,21 @@ function setup_file() {
     export SSL_ALT_KEY_PATH='/config/ssl/key.rsa.pem'
     export SSL_ALT_CERT_PATH='/config/ssl/cert.rsa.pem'
 
-    local DOMAIN='example.test'
     local PRIVATE_CONFIG
+    export DOMAIN_SSL_MANUAL='example.test'
     PRIVATE_CONFIG="$(duplicate_config_for_container .)"
 
     docker run -d --name mail_manual_ssl \
         --volume "${PRIVATE_CONFIG}/:/tmp/docker-mailserver/" \
-        --volume "$(pwd)/test/test-files/ssl/${DOMAIN}/with_ca/ecdsa/:/config/ssl/:ro" \
-        --env DMS_DEBUG=0 \
+        --volume "$(pwd)/test/test-files/ssl/${DOMAIN_SSL_MANUAL}/with_ca/ecdsa/:/config/ssl/:ro" \
+        --env DMS_DEBUG=1 \
         --env SSL_TYPE='manual' \
+        --env TLS_LEVEL='modern' \
```

Comment:
I forgot to mention this in the commit message. `DMS_DEBUG=1` was necessary for the `_notify 'inf'` messages to be seen in the log (I added a test which checks for them). The `TLS_LEVEL` addition was necessary because I call `_setup_ssl` when manual SSL files have changed (and this function will show errors if this env variable is not set).